### DISCUSSION TITLE
Releases and Versioning

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#360] [Ideas]
This well written Repo is distinguished and has a faster pace of new users. It is neat  to have separate releases for Official, Git and Nvidia versions. I'm new to GitHub.

**Comment by T-Crypt** [comment]
@kRHYME7 -- Are you talking about versioning as in like the arch linux AUR? Sort of how packages are "Official", the "-git" version, and the "-nvidia" version? 

Can you explain a little more? I think I understand, but what your referring to is probably getting something like a branch system available for this repo like "Official/Main" branch, maybe a "Dev/Git" branch, and a "Nvidia" branch. 

Correct me if I'm misunderstanding!

**Comment by kRHYME7** [comment]
Yes, that's it! In my case, I'm really having a hard time explaining things when the error comes, unlike you here which explains it well and nicely. 
I guess it will be a branch system with official, git, and Nvidia.  And I will try to keep learning to be nice also. Thanks!

**Reply by T-Crypt** [reply]
A lot of my replies and explanations come from me being in Cyber Security / IT for 8 years now and currently I work at a Managed Service Provider as the Security Advisor and Service Engineer so I'm usually working with customers / clients providing instructions or information!

A lot of my experience is also from ethical hacking and my certifications and just familiarization with linux!  I started on Kali Linux in 2016 and just now switched to Arch this year.

**Reply by kRHYME7** [reply]
Wow !!! That explains a lot. So is it feasible for us to create these branches? And what's the cost of maintaining this, does this consume a lot of work?

**Reply by Gedankenn** [reply]
wow. niceeeee. i'm a embedded devloper. graduated in computer engineering =D

**Comment by prasanthrangan** [comment]
As we are using arch, these packages used are not guaranteed to be stable...
But regarding the config file changes, I'll create a development branch so that it does not break anything in main branch!

**Reply by kRHYME7** [reply]
Thanks! This would be very helpful to some using official hyprland versions.

**Reply by kRHYME7** [reply]
Does the main branch will follow the official Hyprland release? or the Hyprland-git?.

**Reply by Balssh** [reply]
If I'd be to guess main follows the hyprland-git release.
